src/main/resources/db/changelog/changes/06-add-admin-user.yaml

```diff
@@ -0,0 +1,32 @@
+databaseChangeLog:
+  - changeSet:
+      id: add_admin_user
+      author: arsen
+      changes:
+        - insert:
+            tableName: users
+            columns:
+              - column:
+                  name: first_name
+                  value: Admin
+              - column:
+                  name: last_name
+                  value: Adminovich
+              - column:
+                  name: email
+                  value: admin@a.ua
+              - column:
+                  name: password
+                  value: $2a$10$IwQ.C1d7bWTWcFEe/VTR8uqEsEvlgfD6RWmoISmiVc/buQULQv/Hu
+              - column:
+                  name: is_deleted
+                  valueBoolean: false
+        - insert:
+            tableName: user_role
+            columns:
+              - column:
+                  name: user_id
+                  valueNumeric: 1
```

Comment:
use sth like this: `valueComputed: (SELECT id FROM users WHERE email = 'admin@email')`